services/config/src/types.ts

```diff
@@ -6,9 +6,22 @@ type Version = {
     tag?: string
 }
 
+export type DateInput = string | Date | number | null
```

Comment:
I was unsure if we would want to let people initialize a date without passing any arguments (e.g. `fromServerDate()`). I opted yes, so that it aligns with `new Date()` and returns current date/time.